EMIP/1-1000/EMIP00185.xml

```diff
@@ -84,14 +84,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   </incipit>
                 </msItem>
                   <msItem xml:id="ms_i7"><locus from="71r" to="71v"/>
-                  <title type="complete" ref="LIT5888AsmatPrayer"> Asmat prayer against the Tongue of People, on how Jacob said it when he fled from the face of his brother</title>
+                  <title type="complete" ref="LIT1771Lessan"/>
                   <incipit xml:lang="gez">
                     ጸሎት፡ በእንተ፡  ልሳነ፡ ሰብእ፡.
                     በስመ፡ አብ፡ ወወልድ፡... ናሁ፡ ንዜንዎክሙ፡ በዘድኅነ፡ ያዕቆብ፡ እምልሳነ፡ ሰብእ፡ ወጐየ፡ ያዕቆብ፡ እምገጸ፡ ኤሳው፡ እኁሁ፡ ወነገደ፡ ምድረ፡ ሶርያ፡. . .
                     </incipit>
                 </msItem>
                   <msItem xml:id="ms_i8"><locus from="71v" to="78r"/>
-                  <title type="complete" ref="LIT5888AsmatPrayer"> Asmat prayer Against the Tongue of the People, ልሳነ፡ ሰብእ፡ (በእንተ፡ ልሳነ፡ ዘመድ፡ ወባዕድ፡). </title>
+                  <title type="complete" ref="LIT3034RepCh314"/>
```

Comment:
since there are various ways to label the text, if "በእንተ፡ ልሳነ፡ ዘመድ፡ ወባዕድ፡" is the label they have from the MS I would keep it inside the `@title`